src/applications/personalization/dashboard-2/selectors.js

```diff
@@ -0,0 +1,5 @@
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
+export const selectShowDashboard2 = state =>
+  toggleValues(state)[FEATURE_FLAG_NAMES.dashboardShowDashboard2];
```

Comment:
I know I approved a PR that introduced this feature flag name already, but could we change the feature flag to `showDashboard2`?

Reply:
Maybe? We seem to have a rule (unwritten likely!) to try prefacing the feature flag names with the product it is tied to. So for Profile 2 it was something like `profileShowProfile2`. Clearly lots of people don't follow this convention but I kinda like it since the [featureFlagNames](https://github.com/department-of-veterans-affairs/vets-website/blob/c554e452e18121c385812cbe996ff0de3886f970/src/platform/utilities/feature-toggles/featureFlagNames.js) are kept in alpha order it creates at least a tiny level of organization.